Resources/Prototypes/Catalog/VendingMachines/Inventories/chefvend.yml

```
@@ -25,4 +25,6 @@
     FoodBoxDonkpocket: 1
     FoodFrozenSandwich: 2
     FoodFrozenSandwichStrawberry: 2
-    
+    ShrimpBag: 1 #Goob
+  emaggedInventory:
+    ShrimpBag: 5
```

Okay so these fucks are clearly destructive lmao, and too many of them can not only lag you, it can also crash a client apparently. Limiting it to 2 per vendor. More than this should be purchaseable through uplink though.
```suggestion
  emaggedInventory:
    ShrimpBag: 1